### DISCUSSION TITLE
Welcome to amino-coin-generator Discussions!

### DISCUSSION THREAD

**Discussion by ViktorSky** [ViktorSky/amino-coin-generator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fiudy** [comment]
I can't run code.

**Reply by ViktorSky** [reply]
Because? send reasons, error log or screenshot

**Reply by Fiudy** [reply]
I put my account details, but it just keeps asking for more and more accounts to register

**Comment by Fiudy** [comment]
![Captura de tela 2023-08-23 080545](https://github.com/ViktorSky/amino-coin-generator/assets/89918329/5ec35848-6637-42e7-a46e-6f6d496524c5)

**Reply by ViktorSky** [reply]
You need to change the community link because it is expired.

**Comment by Wispwee** [comment]
Hey! Hope it's not too much trouble, but how do I enter my account details into the acc.json?

**Reply by ViktorSky** [reply]
run preparing_json.py

**Comment by TrueAukeyz** [comment]
Hello, I'm unable to run this generator, I don't know if it's because it might be outdated comapred to the app, yet I still send here the this on what I'm struggling with.

I followed the instruction, I executed the preparing_json.py, yet it asks me to reload after installing "k-amino.py". I don't know what it means by reload, but I tried refreshing the explorer, closing and reopening VS Code, and even trying to install k-amino.py by myself by using "pip install k-amino.py", it says I already got the thing.

![Capture d'écran 2024-11-26 220445](https://github.com/user-attachments/assets/b67deacb-4920-46ae-90fd-e1b228bba318)

I also tried to run the main, but I receive yet another error about an extention I downloaded. Which is weird because it said being optional, so why would I receive an error if the code doesn't really need it in the first place...

![Capture d'écran 2024-11-26 220521](https://github.com/user-attachments/assets/142fdaa1-16a1-4b5d-95b4-9e07ea4ec900)

I doubt I get an answer but if there's someone which know how this program work and why I can't make it work, please help me solve the problem or tell me that this generator is outdated. Thanks in advance

**Reply by TrueAukeyz** [reply]
Update, I succeed at making the preparing_json.py by modifying the python version from the one recommanded to the other one proposed (From 3.13.0 to 3.12.7 from Micrcosoft Store), I added three accounts (All three are waiting to be verified) and ran the main file to see if I got the same result than in the video ! And it turns out, I didn't :
![image](https://github.com/user-attachments/assets/624d72be-f624-485d-8e42-e5d67c9a17f9)
One of the two programs works ! It is progress ! Yet still the same problem about the optionnal websocket, but this time, it's not pink !

**Reply by TrueAukeyz** [reply]
Yet another update ! I succeed at making the main program work by downloading miniconda on my laptop ! It work just, fine, I then just need to verify my accounts and it sould work !
![image](https://github.com/user-attachments/assets/78ea7e30-ba81-49d8-a974-ec888afbc094)
In the picture, I pressed CTRL+C to stop the program since it was useless without proper accounts ! The program work just fine !

**Reply by ViktorSky** [reply]
right now the generator is not working, I will update the amino patch soon
